i3pystatus/mail/imap.py

```diff
@@ -1,61 +1,18 @@
 try:
-    from imaplib2 import IMAP4, IMAP4_SSL
+    from imaplib2.imaplib2 import IMAP4, IMAP4_SSL
```

Comment:
Good catch. Looks like a bug in imaplib2: imaplib2/__init__.py, it uses `from imaplib2 import *`, while it should be `from .imaplib2 import *` or `from imaplib2.imaplib2 import *`.

Can you cahgen the "except:" v- to something more specific (like `except ImportError`)?